scripts/ingest.ps1

```
@@ -49,18 +49,56 @@ function Write-ErrorMsg {
+            $process = Get-Process -Id $processId -ErrorAction Stop
+            $actualPath = [System.IO.Path]::GetFullPath($process.Path)
+            $expectedPath = [System.IO.Path]::GetFullPath($FLUENT_BIT_EXE)
+
+            if ([System.StringComparer]::OrdinalIgnoreCase.Equals($actualPath, $expectedPath)) {
                 return $true
             }
         }
         catch {
-            return $false
+            # Treat missing or inaccessible processes as stale.
         }
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)